app/models/service_template.rb

```diff
@@ -10,6 +10,20 @@ class ServiceTemplate < ApplicationRecord
     "storage"         => _("Storage")
   }.freeze
 
+
+  CATALOG_ITEM_TYPES = {
```

Comment:
I think that more that just one UI might need to have a nice name for the catalog item. Also it does not make sense for me for the UIs to track what types of catalog item the backend has.

So I'd say it should stay on the backend side.

Note: although it need not look like that now, the values might need i18n care. Ping @mzazrivec